src/unix.rs

```diff
@@ -356,3 +359,8 @@ pub fn unlock(fd: FileDesc) -> Result<(), Error> {
 pub fn close(fd: FileDesc) {
     unsafe { libc::close(fd) };
 }
+
+/// An uninitialized file descriptor
+pub(crate) fn uninitialized_fd() -> FileDesc {
+    -1
+}
```

Comment:
Again, I'd rather not have this.

Reply:
It's not callable outside this crate (`pub(crate)` vs `pub`)